Reorganize SharePoint Embedded docs (#10941)
SharePoint/sp-dev-docs: docs/embedded/admin/admin-overview.md

---
title: Admin Overview
description: Learn how administrators manage SharePoint Embedded apps, containers, billing, and compliance in Microsoft 365.
ms.date: 07/10/2026
ms.reviewer: shsaravanan
ms.localizationpriority: high
---

# Admin Overview

> **Applies to:** Consuming tenant admin — SharePoint Embedded admin / Global admin / SharePoint admin / Compliance admin / Security admin

Reviewer: ```suggestion
**Applies to:** Consuming tenant admin — SharePoint Embedded admin / Global admin / Compliance admin / Security admin
```

Organizations use SharePoint Embedded apps to store files and documents in containers that inherit Microsoft 365 security, compliance, and administration controls.
Administrators manage those apps and containers through the SharePoint admin center and SharePoint Online Management Shell.

Use this overview to decide which administrator role and management surface to use.
Then follow the linked how-to articles for app creation, installation, billing, container lifecycle, and compliance operations.

> [!IMPORTANT]
> Assign the **SharePoint Embedded Administrator** role before delegating SharePoint Embedded administration.
> Global Administrators already have the permissions of this role, but should use least-privileged delegation for daily operations.

## Admin roles

SharePoint Embedded administration commonly involves the following roles.

| Role | Use it for |
| --- | --- |
| Global Administrator | Assign the SharePoint Embedded Administrator role and perform any SharePoint Embedded admin task when needed. |
| SharePoint Embedded Administrator | Manage SharePoint Embedded apps and containers through SharePoint admin center and supported SharePoint PowerShell cmdlets. |

> | SharePoint administrator | Manage SharePoint service configuration. This role doesn't administer SharePoint Embedded apps or containers; use the SharePoint Embedded Administrator role for those tasks. |

Reviewer: This should be added as a note instead since it's not part of the relevant roles.

```suggestion
```

| Tenant administrator | Manage apps and settings in the consuming Microsoft 365 tenant. |
| Compliance administrator | Configure Microsoft Purview audit, retention, DLP, eDiscovery, and related policies. |
| Security administrator | Configure security policies and investigate security events that apply to SharePoint Embedded content. |

The SharePoint Embedded Administrator role is available in Microsoft Entra and the Microsoft 365 admin center.
It is dedicated to SharePoint Embedded administration.

> It does not grant regular SharePoint site management access.

Reviewer: ```suggestion
It doesn't grant regular SharePoint site management access.
```

> For example, a SharePoint Embedded Administrator does not see the **Active sites** or **Deleted sites** pages in the SharePoint admin center and cannot run site-specific SharePoint PowerShell cmdlets.

Reviewer: ```suggestion
For example, a SharePoint Embedded Administrator doesn't see the **Active sites** or **Deleted sites** pages in the SharePoint admin center and cannot run site-specific SharePoint PowerShell cmdlets.
Likewise, the SharePoint administrator role doesn't administer SharePoint Embedded apps or containers.
```
